lib/Dialect/Synth/SynthOps.cpp

```diff
@@ -39,9 +39,9 @@ llvm::APInt MajorityInverterOp::evaluate(ArrayRef<APInt> inputs) {
          "Number of inputs must match number of operands");
 
   if (inputs.size() == 3) {
```

Comment:
Could you keep the previous implementation? `evaluate` is meant to evaluate the op based on input values so it's necessary to look at inversion.